src/bindings/python/flux/job.py

```diff
@@ -258,6 +258,11 @@ def job_list(flux_handle, max_entries=0, attrs=[], userid=os.geteuid(), states=0
     return JobListRPC(flux_handle, "job-info.list", payload)
 
 
+def job_list_inactive(flux_handle, since, max_entries=1000, attrs=[]):
```

Comment:
Would it make sense to have a default of `since=0.`? That way `job_list_inactive(flux_handle)` returns the 1000 most recently inactive jobs. It is only a minor convenience, so not a big deal...